be/CMakeLists.txt

```diff
@@ -55,11 +55,13 @@ set(BASE_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
 set(ENV{STARROCKS_HOME} "${BASE_DIR}/../")
 set(BUILD_DIR "${CMAKE_CURRENT_BINARY_DIR}")
 set(THIRDPARTY_DIR "$ENV{STARROCKS_THIRDPARTY}/installed/")
+set(STARROCKS_THIRDPARTY "$ENV{STARROCKS_THIRDPARTY}")
```

Comment:
Why need set this env variable ?

Reply:
Clion IDE depends on it.

Reply:
Specifically, in Clion remote development mode.